rust/crd/src/lib.rs

```diff
@@ -84,7 +84,7 @@ pub const LDAP_BIND_PASSWORD: &str = "ldap.bind-password";
 pub const LDAP_USER_BASE_DN: &str = "ldap.user-base-dn";
 pub const LDAP_GROUP_AUTH_PATTERN: &str = "ldap.group-auth-pattern";
 pub const LDAP_ALLOW_INSECURE: &str = "ldap.allow-insecure";
-pub const LDAP_SSL_TRUST_CERTIFICATE: &str = "ldap.ssl-trust-certificate";
+pub const LDAP_SSL_TRUST_STORE_PATH: &str = "ldap.ssl.truststore.path";
```

Comment:
1. Did the old property break anything?
2. Looks like this was added in [Trino 376](https://trino.io/docs/current/release/release-376.html?highlight=ldap%20ssl%20truststore%20path), do we still support any versions older than that?

Reply:
Yeah was added in 376 (our oldest is 377). In the doc examples (even for the newer versions like 395), they used the trust certificate but this broke 377 (no problem with newer versions strangely...).

Reply:
I did not investigate further but will start another custom test once the actions are done.

Reply:
Ok, makes sense then.